apps/meteor/client/views/room/components/body/composer/messageBox/MessageBox.tsx

```diff
@@ -391,11 +431,25 @@ const MessageBox = ({
 					aria-activedescendant={ariaActiveDescendant}
 				/>
 				<div ref={shadowRef} style={shadowStyle} />
+				{showMarkdownPreview && <MessagePreview md={md} channels={channels} mentions={mentions} />}
+
+				{IsMessagePreviewAllowed && (
```

Comment:
The IconButton component is conditionally rendered when user preference `IsMessagePreviewAllowed` is `true`.